project/src/components/main/main.jsx

```diff
@@ -1,16 +1,23 @@
-import React from 'react';
+import {React, useState} from 'react';
 import PropTypes from 'prop-types';
 import 'leaflet/dist/leaflet.css';
 import Offers from '../offers/offers';
 import offerProps from '../offers/offer.props';
 import Map from '../../components/map/map';
 
-function WelcomeScreen(props) {
-  const { offers } = props;
+function WelcomeScreen({offers, getSelectedPoint}) {
+
+  const [selectedPoint, setSelectedPoint] = useState({});
+
+  const listHoverHandler = function (offer) {
+    setSelectedPoint(offer);
```

Comment:
```suggestion
  const listHoverHandler = function (offer) {
```
Стейт не нужен. Локальные колбеки компонента названы через handle